GitExtUtils/GitArgumentBuilder.cs

```diff
@@ -177,8 +178,14 @@ public override string ToString()
 
             // 7 = "-c " + "=" + " " + 2 (for possible quotes of value)
             var capacity = _configItems.Sum(i => i.Key.Length + i.Value.Length + 7) + _command.Length + 1 + arguments.Length;
+            capacity += (_gitArgs?.Length ?? -1) + 1;
 
             var str = new StringBuilder(capacity);
+            if ((_gitArgs?.Length ?? 0) > 0)
```

Comment:
`_gitArgs?.Length ?? ` construct is used twice, worth reconciling it once then reuse the result

```
var gitArgsLength = _gitArgs?.Length ?? -1;
capacity += gitArgsLength + 1;

if (gitArgsLength > 0)
{
    ...
```